Lib/ufo2ft/featureWriters/kernFeatureWriter.py

```diff
@@ -386,7 +386,8 @@ def _makeKerningLookup(self, name, ignoreMarks=True):
         lookup = ast.LookupBlock(name)
         if ignoreMarks and self.options.ignoreMarks:
             # We only want to filter the spacing marks
-            marks = self.context.gdefClasses.mark
+            marks = set(self.context.gdefClasses.mark or []) & set(self.context.glyphSet.keys())
```

Comment:
should we do this intersection by the time we actually set these gdefClasses to begin with? I think we know the glyphSet at that stage. Then all downstream users can simply take that for granted, that non export glyphs have been excluded from the gdefClasses

Reply:
Yeah, it's a bit weird that `getOpenTypeCategories` includes glyphs which aren't in the eventual font. That would be cleaner but I was trying for minimally invasive changes.

Reply:
it's ok like this. If you can blacken and merge, I'd like to push a small release out today